src/test/shell/bazel/bazel_build_event_stream_test.sh

```diff
@@ -49,7 +75,7 @@ function test_fetch_test() {
   # build-event stream.
   bazel clean --expunge
   rm -f "${TEST_log}"
-  bazel fetch --build_event_text_file="${TEST_log}" //pkg:main \
```

Comment:
Is this for preventing MSYS from translating them as path? Is setting 
```
export MSYS_NO_PATHCONV=1
export MSYS2_ARG_CONV_EXCL="*"
```
a better way?

Reply:
Yes, that's the reason.
Yes, let me try that.